docs/source/adavanced_guides/index.rst

```diff
@@ -0,0 +1,25 @@
+=============
+Advanced Guides
+=============
```

Comment:
I was wondering if we should we put this in advanced guides section? If we put it in a normal guides section it would signal that GPU is something Yatai has first class support for.

Reply:
I think GPU deployment is usually an advanced topic.

Reply:
Ideally it shouldn't be right? fractionaly gpu is an advanced topic but generic GPU setup should be first class I feel

Reply:
It's hard to give this section a name for me. We would change It to a better one once we come up with an idea.